src/core/meshcop/commissioner.cpp

```diff
@@ -170,7 +170,7 @@ otError Commissioner::Start(otCommissionerStateCallback  aStateCallback,
     SetState(OT_COMMISSIONER_STATE_PETITION);
 
 exit:
-    if (error != OT_ERROR_NONE)
+    if (error != OT_ERROR_NONE && error != OT_ERROR_ALREADY)
```

Comment:
Good catch. Can we, however, apply this if statement for CoapSecure.Stop() method and leave logs as they are now both in start and stop?

Reply:
Don't have strong opinion on removing the log for `OT_ERROR_ALREADY`.  
But if we are treating `OT_ERROR_ALREADY` same as success, should we better keep this rule even in logging?

Reply:
Also don't have very strong opinion, but leaning towards leaving the log, so while debugging we know someone tried to "rerun" commissioner.

Reply:
Log kept for `OT_ERROR_ALREADY`